CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+## v1.1.0
+
 Added:
  * Add unstable `otel_metrics_ingestion` resource.
```

Comment:
As part of moving trace_tail_sampling_rules to genericResource, I skipped the changelog since it was more of an internal refactor, but it does have a small customer-facing change: it enables dry-run support. It may be worth mentioning here,

 * Add server-side validation support for `chronosphere_trace_tail_sampling_rules`